### PR TITLE
added complete project

### PR DESCRIPTION
**Pull request by 1khansaad**

### CODE REVIEW

File: cart.html [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+                            </div>
+                        </div>
+                    </td>
+                    <td><input type="number" value="1"></td>
+                    <td>$50.00</td>
+                </tr>
+                <tr>
+                    <td>
+                        <div class="cart-info">
+                            <img src="images/buy-3.jpg" alt="">
```
Comment (Abbas4tech) [inline cart.html:75]: add alt @1khansaad

File: cart.html [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+                    </tr>
+                </table>
+            </div>
+        </div>
+
+       
+
+        <!-- footer -->
+
+        <div class="footer">
```
Comment (Abbas4tech) [inline cart.html:110]: use `footer` tag

File: Product.html [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+                            <i class="bi bi-star-fill"></i>
+                            <i class="bi bi-star-fill"></i>
+                            <i class="bi bi-star-fill"></i>
+                            <i class="bi bi-star-half"></i>
+                    </div>
+                    <p>$50.00</p>
+                </div>
+            </div>
+            <div class="col-4">
+                <img src="images/product-9.jpg" alt="product-9">
```
Comment (Abbas4tech) [inline Product.html:160]: Place proper alt text since `alt="product-9"` doesn't specify the description about image

File: Product.html [hunk 4]
```diff
@@ -0,0 +1,283 @@
...
+                <span>2</span>
+                <span>3</span>
+                <span>4</span>
+                <span>&#8594</span>
+            </div>
+        </div>
+
+        <!-- footer -->
+
+        <div class="footer">
```
Comment (Abbas4tech) [inline Product.html:228]: use `footer` tag

File: style.css [hunk 5]
```diff
@@ -0,0 +1,508 @@
...
+  color: #555;
+}
+.container nav a:hover {
+  color: red;
+  transition: 0.2s;
+}
+.container {
+  max-width: 1300px;
+  margin: auto;
+  padding: 0 25px 0 25px;
```
Comment (Abbas4tech) [inline style.css:40]:
```css
padding : 0 25px;
```

**Review by Abbas4tech — COMMENTED** [review]
Use short hands

File: style.css [hunk 6]
```diff
@@ -0,0 +1,508 @@
...
+  margin: 30px 0;
+
+  background: #ff523b;
+  color: white;
+  border-radius: 30px;
+  transition: 0.4s;
+}
+.btn:hover {
+  background: red;
+  color: black;
```
Comment (Abbas4tech) [inline style.css:76]:
```css
color: white;
```

Comment (Abbas4tech) [reply style.css:76]: Color this to white so it will look decent @1khansaad

**Review by Abbas4tech — COMMENTED** [review]
improve color

File: Product.html [hunk 7]
```diff
@@ -0,0 +1,283 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.7.2/font/bootstrap-icons.css">
+    <title>Redstore Ecommers web design</title>
+</head>
+<body>
+        <div class="container">
+            <div class="navbar">
```
Comment (nasir-k) [inline Product.html:14]: You can use nav(semantic)tag instead of giving class or I'd to a div .

File: Product.html [hunk 8]
```diff
@@ -0,0 +1,283 @@
...
+                            <i class="bi bi-star-fill"></i>
+                            <i class="bi bi-star-fill"></i>
+                            <i class="bi bi-star-fill"></i>
+                            <i class="bi bi-star-half"></i>
+                    </div>
+                    <p>$50.00</p>
+                </div>
+            </div>
+            <div class="col-4">
+                <img src="images/product-10.jpg" alt="product-10">
```
Comment (nasir-k) [inline Product.html:174]: In "alt" attribute always write short discription about the image like black shoes or like that ..

File: Product.html (hunk 4 above)
Comment (nasir-k) [inline Product.html:228]: Use semantics

File: cart.html [hunk 9]
```diff
@@ -0,0 +1,170 @@
...
+            <table class="tb-1">
+                <tr>
+                    <th>Product</th>
+                    <th>Quantity</th>
+                    <th>Subtotal</th>
+                </tr>
+                <tr>
+                    <td>
+                        <div class="cart-info">
+                            <img src="images/buy-1.jpg" alt="">
```
Comment (nasir-k) [inline cart.html:45]: Add discription in alt

**Pull request merged**